datahub-web-react/src/graphql/glossaryTerm.graphql

```diff
@@ -1,9 +1,33 @@
-query getGlossaryTerm($urn: String!) {
+query getGlossaryTerm($urn: String!, $start: Int!, $count: Int!) {
```

Comment:
hey lal- looks like adding these required params is causing build failures:

https://github.com/linkedin/datahub/pull/3103/checks?check_run_id=3353708342.

I would either remove them as required params (they are optional params to relationships() type) or populate them from react in this PR

Reply:
yup, done